ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v7-presubmits.yaml

```diff
@@ -237,6 +237,24 @@ presubmits:
         resources: {}
       serviceAccountName: ci-operator
     trigger: (?m)^/test( | .* )e2e-scaleupdown-v7.0,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-v7
+    context: e2e-scaleupdown-v7.1
+    decorate: true
+    name: pull-ci-azure-release-v7-e2e-scaleupdown-v7.1
```

Comment:
Do we need this on? O_o

Reply:
Probably doesn't hurt? Do we want to also add a 7.1 upgrade case?

Reply:
we already have one `pull-ci-azure-release-v7-upgrade-v7.0`